docker/stolon-development/Dockerfile

```diff
@@ -9,6 +9,15 @@ RUN set -x \
       && git checkout 71d109541f800fcacb2d8e3222715d84e569ccb5 \
       && go build -o stolon-keeper cmd/keeper/main.go
 
+# GoCardless runs this fork for PgBouncer metrics. We'll likely change this in
```

Comment:
while you're here, it'd be sweet to install all the gocardless versions of the stolon components here, just in case we make a change in another component and wonder why it doesn't take effect in docker-compose.

Reply:
Yeah that makes sense, can change that now.